src/DbgHelp/DbgHelp+LOADED_IMAGE.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="LOADED_IMAGE"/> nested type.
+    /// </content>
+    public static partial class DbgHelp
+    {
+        /// <summary>
+        /// Contains information about the loaded image.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct LOADED_IMAGE
+        {
+            /// <summary>
+            /// The file name of the mapped file.
+            /// </summary>
+            [MarshalAs(UnmanagedType.LPStr)]
+            public string ModuleName;
+
+            /// <summary>
+            /// A handle to the mapped file.
+            /// </summary>
+            public IntPtr hFile;
+
+            /// <summary>
+            /// The base address of the mapped file.
+            /// </summary>
+            public IntPtr MappedAddress;
+
+            /// <summary>
+            /// A pointer to an IMAGE_NT_HEADERS structure.
```

Comment:
Ya. hFile and perhaps others may eventually be exposable via safe handles, so they can be passed to our other methods. But I haven't taken the time yet to prove that .NET won't prematurely close the file handle on us or something weird like that. Also, there are several very deep structs that we can and eventually should define that right now are just IntPtr's here.
These structs and methods are defined *so badly* in MSDN. Pointers are typed as PUCHARs, such that the automatic P/Invoke method generation produces "byte" or 'string' as the type for the field, which led to crashes.